### PR TITLE
add tipograf@1.0.2 w/ npm auto-update

### LINKED ISSUE #9231: [Request] Add tipograf
#**Library name:** tipograf
**Git repository url:** https://github.com/mightyCrow/tipograf
npm : https://www.npmjs.com/package/tipograf
**npm package url(optional):** tipograf

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #9231 
@x09326 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/mightyCrow/tipograf
 * Official website (optional, not the repository): https://mightycrow.github.io/tipograf/
 * NPM package url (optional): https://www.npmjs.com/package/tipograf
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 96
   - Count of watchers: 4
   - Count of forks: 8
 * NPM download stats (optional):
   - Downloads in the last day: 6
   - Downloads in the last week: 80
   - Downloads in the last month: 105

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9231, cc@mightyCrow

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Ping

**Comment by kennynaoh** [comment]
ping @x09326

File: ajax/libs/tipograf/1.0.2/tipograf.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
+/**
+ * tipograf - Lightweight CSS typography base for your next project
+ * @version v1.0.1
```
Comment (kennynaoh) [inline ajax/libs/tipograf/1.0.2/tipograf.css]: @x09326 
I think the author just forgot to update the version number.
I found that there are some updates in `tipograf.css` in `v1.0.2` and also updated the version in `package.json`
https://github.com/mightyCrow/tipograf/commit/5e22c14171772b40fa8948b79a2ba088fbf48973

**Comment by x09326** [comment]
@kennynaoh 
Okay.
Please rebase with the latest branch.

**Comment by kennynaoh** [comment]
@x09326 
I have rebased to the latest master br.
Please help me review it again.
Thanks!

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

File: ajax/libs/tipograf/1.0.2/tipograf.css (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/tipograf/1.0.2/tipograf.css]: @x09326 we usually rely on the git tag and npm version, so just make sure we added the same as upstream, if so, this won't be a problem if the content contains the wrong version number.

**Pull request merged**